a3p-integration/proposals/a:upgrade-next/post.test.js

```diff
@@ -0,0 +1,14 @@
+import test from 'ava';
+
+import { agd } from '@agoric/synthetic-chain/src/lib/cliHelper.js';
+
+test('Ensure MaxBytes param was updated', async t => {
+  const { value: rawParams } = await agd.query(
+    'params',
+    'subspace',
+    'baseapp',
+    'BlockParams',
+  );
+  const blockParams = JSON.parse(rawParams);
+  t.is(blockParams.max_bytes, '5242880');
+});
```

Comment:
Oh this probably should be removed since it should be upstream already (it's not testing this simulated upgrade anymore)